### PR TITLE
streamlit slightly less ugly

### LINKED ISSUE #5: Visualization not great/ Streamlit is ugly
I don't like the visualization. let's fix that. Also the streamlit app is ugly

### PR DESCRIPTION
**Pull request by devyn-miller**
hopefully these changes make the app a bit more visually appealing :)

### CODE REVIEW

**Comment by tristntran** [comment]
It's a nice improvement, but doesn't work well for dark mode. Also maybe we need to refine the issue. for more specific goals.

File: src/main.py [hunk 1]
```diff
@@ -4,38 +4,103 @@
...
-    """
-    # Create the plot
     fig, ax = plt.subplots()
-    ax.bar(np.arange(len(current_state)), current_state)
-    ax.set_title('Current State of the Gambler\'s Ruin')
-    ax.set_xlabel('Cash')
+    colors = plt.cm.viridis(current_state / current_state.max())
+    bars = ax.bar(np.arange(len(current_state)), current_state, color=colors)
+    ax.set_title("Current State of the Gambler's Ruin")
+    ax.set_xlabel('Cash Amount')
```
Comment (tristntran) [inline src/main.py:94]: X axis label is incorrect. we should use the state map.

File: src/main.py [hunk 2]
```diff
@@ -4,38 +4,103 @@
 from gamblers_ruin import (create_transition_matrix, find_nth_state, 
                            find_expected_value, run_gamblers_ruin)
 
-# Set the title of the app 
+# more expansive page config
+st.set_page_config(page_title="Gambler's Ruin Simulator", layout="wide")
+font = "sans serif"
+
+# Theme switch for light or dark mode
+theme = st.sidebar.radio("Choose Font Color", ('Default', 'Red', 'Green', 'Blue'))
```
Comment (tristntran) [inline src/main.py:12]: Can we move the theme radio button to the bottom of the sidebar?

**Review by tristntran — APPROVED** [review]
Good intermediate progress. I would suggest merging this and adding more updates. Of note, I think a plotly graph would suit the situation well. Also, you can configure streamlit themes using the configuration style hocon files

https://docs.streamlit.io/knowledge-base/using-streamlit/path-streamlit-config-toml

https://docs.streamlit.io/library/advanced-features/configuration#set-configuration-options

**Comment by ProvenNoIQ** [comment]
Going to need to add slider functionality for period but overall lgtm

**Review by ProvenNoIQ — APPROVED** [review]

**Review by jordanbruce44 — APPROVED** [review]
Still doesn't run in all cases, but since it's only affecting the visualization of the application, lgtm

**Pull request merged**